salt/modules/win_system.py

```diff
@@ -65,24 +74,32 @@ def _convert_date_time_string(dt_string):
     return dt_obj.strftime('%Y-%m-%d %H:%M:%S')
 
 
+def _to_unicode(instr):
+    '''
+    Converts from current users character encoding to unicode.
+    When instr has a value of None, the return value of the function
+    will also be None.
+    '''
+    if instr is None or isinstance(instr, six.text_type):
+        return instr
+    else:
+        return six.text_type(instr, 'utf8')
```

Comment:
I think is `utf-8` and not `utf8`.

But, more importantly, shouldn't we be using `__salt_system_encoding__` for these? This isn't preserving the user's system setting.